bazel/api_binding.bzl

```diff
@@ -13,6 +13,7 @@ def _default_envoy_api_impl(ctx):
     ]
     for d in api_dirs:
         ctx.symlink(ctx.path(ctx.attr.envoy_root).dirname.get_child(ctx.attr.reldir).get_child(d), d)
+    ctx.symlink(ctx.path(ctx.attr.envoy_root).dirname.get_child("bazel").get_child("utils.bzl"), "utils.bzl")
```

Comment:
ive tried a few different ways to make the `json_data` macro available in the api part of the repo

not sure the best way

Reply:
Possibly the cleanest way is to put the `json_data` macro inside `api/bazel`. That makes the dependency chain simpler.

Reply:
not sure why i didnt think of that - ill move it

Reply:
...in a follow up...